src/sentry/utils/pytest/selenium.py

```diff
@@ -337,11 +337,12 @@ def snapshot(self, name, mobile_only=False):
                             "window.__closeAllTooltips && window.__closeAllTooltips()"
                         )
 
-            with self.mobile_viewport():
+            if not desktop_only:
```

Comment:
@billyvg Please is this change fine for you?

Reply:
imo we should include the mobile snapshots, it’s not only for mobile devices -- it applies to desktop devices on smaller window sizes as well